addons/go.mod

```diff
@@ -36,7 +36,7 @@ require (
 	k8s.io/utils v0.0.0-20220812165043-ad590609e2e5
 	knative.dev/pkg v0.0.0-20220302134643-d2cdc682d974
 	sigs.k8s.io/cluster-api v1.2.8
-	sigs.k8s.io/cluster-api-provider-vsphere v1.5.0
+	sigs.k8s.io/cluster-api-provider-vsphere v1.5.2
```

Comment:
Why is it 1.5.2 here?

Reply:
Addressed, thanks.
Previously this same PR was bumping CAPV to 1.5.2, updated it to point to v1.5.3. Missed one of the imports, thanks!